generator.yaml

```diff
@@ -19,6 +19,8 @@ operations:
       RetainAllVariantProperties: true
   DeleteEndpoint:
     custom_implementation: customDeleteEndpoint
+  DeleteModelPackageGroup:
```

Comment:
nit: check if you can use a hook in sdkDelete instead of this method

this is old way